src/core/layout/scrollable-list/scrollable-list.js

```diff
@@ -0,0 +1,94 @@
+/**
```

Comment:
Just an observation - but does everything in here need to be exported?
I recently edited the catalogue as there was only one thing needed exporting.

We tend to only export what is needed - exports being the public interface and eveything else being private.

If all this is neede though that's fine :)

Reply:
Fixed in today's refactor- now it's only exporting the interface.